frame/bounties/src/tests.rs

```diff
@@ -1122,3 +1168,37 @@ fn accept_curator_handles_different_deposit_calculations() {
 		assert_eq!(Balances::reserved_balance(&user), expected_deposit);
 	});
 }
+
+#[test]
+fn propose_bounty_works_second_instance() {
+	new_test_ext().execute_with(|| {
+		System::set_block_number(1);
+
+		Balances::make_free_balance_be(&Treasury1::account_id(), 101);
+		assert_eq!(Treasury1::pot(), 100);
+
+		assert_ok!(Bounties1::propose_bounty(Origin::signed(0), 10, b"1234567890".to_vec()));
+
+		assert_eq!(last_event_second_instance(), BountiesEvent::BountyProposed { index: 0 });
+
+		let deposit: u64 = 85 + 5;
+		assert_eq!(Balances::reserved_balance(0), deposit);
+		assert_eq!(Balances::free_balance(0), 100 - deposit);
+
+		assert_eq!(
+			Bounties1::bounties(0).unwrap(),
+			Bounty {
+				proposer: 0,
+				fee: 0,
+				curator_deposit: 0,
+				value: 10,
+				bond: deposit,
+				status: BountyStatus::Proposed,
+			}
+		);
+
+		assert_eq!(Bounties1::bounty_descriptions(0).unwrap(), b"1234567890".to_vec());
+
+		assert_eq!(Bounties1::bounty_count(), 1);
+	});
```

Comment:
Similarly, you should show `Treasury1` is affected here, and `Treasury` is not.

Then we can show for certain that these two pallets are correctly connected.